scylla-server-rust/src/main.rs

```diff
@@ -186,6 +198,11 @@ async fn main() {
         )
         // SYSTEMS
         .route("/systems", get(system_controller::get_all_systems))
+        // CONFIG
+        .route(
+            "/config/:key",
+            post(send_config_controller::send_config).layer(Extension(client)),
```

Comment:
can you rename this, 'config/set/:key', it just makes the intention that this is supposed to a post a little more clear.

Also, IMO key should be changed to configName, or configKey if key is the name we're using everywhere else